### ISSUE TITLE
Toogle and switches

### ISSUE THREAD

**Issue by StreiffBerenice** [Orange-OpenSource/Orange-Boosted-Bootstrap#172]
@liyokuna COLOUR CONTRAST: put for example black on white instead of white on orange background DEFAULT STATE OF CHECKBOXES : the default state is "checked" instead of " unchecked" See the components [toogles and switches](https://boosted.orange.com/docs/4.3/boostwatch/#switches).

**Comment by ffoodd** [comment]
Also:
* [ ] remove `o-switch` → [Orange Switches](https://boosted.orange.com/docs/4.3/components/forms/#orange-switches)
* [ ] test and improve Bootstrap switches?

**Comment by ffoodd** [comment]
Orange Switches will be dropped in v5 and are marked as deprectaed in v4. See #283.

FWIW, colors are defined by Digital Brand Guidelines, and default state is not component-side: only implementation that can change for each use case.
